docs/concepts/deployments.md

```diff
@@ -167,9 +167,9 @@ Every step can optionally provide a `requires` field that Prefect will use to au
 
     This capability is useful with multiple deployments that require different deployment instructions.
 
-For more information on the mechanics of steps, [see below](#deployment-mechanics).
+See below for more information on the mechanics of steps!
```

Comment:
I'm not fully convinced we need this line at all. Thoughts?

Reply:
Yeah I don't think we need this line at all

Reply:
Sounds good I'll remove!